client/me/purchases/components/loading-placeholder/index.jsx

```diff
@@ -21,15 +22,20 @@ class LoadingPlaceholder extends React.Component {
 	static propTypes = {
 		path: PropTypes.string,
 		title: PropTypes.string.isRequired,
+		isSiteLevelBilling: PropTypes.bool.isRequired,
 	};
 
 	goBack = () => {
 		page.back( this.props.path || '/' );
 	};
 
 	render() {
+		const classes = classnames( 'loading-placeholder', {
+			'is-wide-layout': this.props.isSiteLevelBilling,
```

Comment:
There may be a way to do this without making this component aware of the difference between site-level billing and account-level billing. If the only difference between the two is a className, it seems unnecessary to add the whole concept of "which type of billing section is this" to this component.

Instead, I suggest making it possible to pass `className` in as a prop to this component and merge that prop with `loading-placeholder`. Then you can set it to `"is-wide-layout"` from `AddCardDetails` and `EditCardDetails` without having to expose any idea of location.

As for those components, there may be an indirect way to also set the className of the placeholder. I suggest that instead of `isSiteLevelBilling` there, we use something like `placeholderClassName`. (If there was something else that was different for site-level billing in those components, that prop would seem to have more justification for me, but there is not.)

Reply:
It's funny you say that, I initially named the prop isFullWidth. I think I would prefer that to passing a class name which is also another way to go but I don't think we need that kind of flexibility. What do you think of `isFullWidth`?

Reply:
All cleaned up!